R/plotconfiguration.R

```diff
@@ -333,6 +333,11 @@ ResVsPredPlotConfiguration <- R6::R6Class(
   inherit = PlotConfiguration
 )
 
+#' @title ResVsTimePlotConfiguration
+#' @description R6 class defining the configuration of a `ggplot` object for Res vs Pred/Time plots
+#' @export
+ResVsTimePlotConfiguration <- ResVsPredPlotConfiguration

```

Comment:
same question as for `ResVsTimeDataMapping `

Reply:
I don't think this is right for class. Please create another class that derives from this one instead.